lib/bolt/transport/local.rb

```diff
@@ -83,14 +83,24 @@ def run_task(target, task, arguments, _options = {})
         implementation = task.select_implementation(target, PROVIDED_FEATURES)
         executable = implementation['path']
         input_method = implementation['input_method'] || 'both'
-
-        # unpack any Sensitive data, write it to a separate variable because
-        # we log 'arguments' below
-        unwrapped_arguments = unwrap_sensitive_args(arguments)
-        stdin = STDIN_METHODS.include?(input_method) ? JSON.dump(unwrapped_arguments) : nil
-        env = ENVIRONMENT_METHODS.include?(input_method) ? envify_params(unwrapped_arguments) : nil
+        extra_files = implementation['files']
 
         with_tmpscript(executable, target.options['tmpdir']) do |script, dir|
+          if extra_files
```

Comment:
can we create an installdir variable first. I think that would make this slightly more readable.